lib/OpenQA/Schema/Result/ScheduledProducts.pm

```diff
@@ -429,16 +430,17 @@ sub _schedule_iso {
 
 =over 4
 
-=item _settings_key()
+=item _job_ref()
 
-Return settings key for given job settings. Internal method.
+Return the "job reference" for the specified job settings. It is used internally as a key for the job in various
+hash maps. It is also used to refer to a job in dependency specifications.
 
 =back
 
 =cut
 
-sub _settings_key ($settings) {
-    my ($test, $machine) = ($settings->{TEST}, $settings->{MACHINE});
+sub _job_ref ($job_settings) {
```

Comment:
This rename confused me a bit. Maybe not that important, but "key" is more like what I expect here. This doesn't "refer" to anything.